pkg/resource/aws/testdata/acc/aws_apigatewayv2_integration/terraform.tf

```diff
@@ -0,0 +1,46 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+terraform {
+  required_providers {
+    aws = "3.19.0"
+  }
+}
+
+resource "aws_apigatewayv2_api" "test" {
+    name          = "test"
+    protocol_type = "HTTP"
+    body = jsonencode({
+        openapi = "3.0.1"
```

Comment:
May be we should add a test for openapi v2 as well?

Reply:
it's not the purpose of this pr this will only test the integration.  but yeah we might want to add some test for embedded api in acceptance tests